pyls/_utils.py

```diff
@@ -198,3 +199,7 @@ def is_process_alive(pid):
             return e.errno == errno.EPERM
         else:
             return True
+
+
+def camel_to_underscore(camelcase):
+    return re.sub('([A-Z]+)', r'_\1', camelcase).lower()
```

Comment:
I think this would rewrite `HelloWorld` to `_hello_world`. Can we be more specific, e.g. this: https://stackoverflow.com/questions/1175208/elegant-python-function-to-convert-camelcase-to-snake-case